frontend/src/router.ts

```diff
@@ -32,7 +32,8 @@ function createRouter() {
   let marketRoutes: RouteConfig[] = [];
   if(featureFlagMarket) {
     marketRoutes = [
-      { path: '/market', name: 'market', component: Market }
+      { path: '/market/:category/:type/:id', name: 'marketSearch', component: Market },
```

Comment:
this should really all be query params on /market. we have a lot of search criteria to go through.